### DISCUSSION TITLE
I can't import the project in eclipse

### DISCUSSION THREAD

**Discussion by gsierra-spiria** [mapstruct/mapstruct#3302] [Q&A]
Hi everyone, I hope somebody can give me and advice.
I'm configuring the dev env for mapstruct to see if I can help with some easy tickets (for now). I've cloned the repo but I was not able to import and build the project successfully in eclipse, In `MappingProcessor` I'm having issues with `import org.mapstruct.ap.internal.gem.MapperGem;` because the class cannot be found, Am I missing something when importing the project?

**Comment by gsierra-spiria** [comment]
My apologies, my eclipse instance didn't have installed `m2e-apt`. It's not easy to find it 😅

**Reply by filiphr** [reply]
This is in our README https://github.com/mapstruct/mapstruct#eclipse 😄
